### PR TITLE
Feature/nav categories

### LINKED ISSUE #119: Mostrar categorías principales en la navegación
En el dropdown de categorías en la navegación principal, cargar las categorías principales de la base de datos y mostrarlas allí con sus respectivos hipervínculos

### PR DESCRIPTION
**Pull request by fdsosa**
Resolved #119

### CODE REVIEW

File: template_dev/components/footer/footer.php [hunk 1]
```diff
@@ -4,9 +4,18 @@
       <div>
         <h3>Categorías</h3>
         <ul class="links">
-          <li><a href="#">Categoría 1</a></li>
-          <li><a href="#">Categoría 2</a></li>
-          <li><a href="#">Categoría 3</a></li>
+          <?php if (count(getGlobal('categories')) > 0) : ?>
+            <?php 
+              for($i = 0; $i < 3; $i++) {
```
Comment (Miguelslo27) [inline template_dev/components/footer/footer.php]: We should keep all categories here, as in the header navigation (I know, it breaks the layout in the footer, we can record a bug later)

File: template_dev/components/footer/footer.php [hunk 2]
```diff
@@ -4,9 +4,18 @@
...
+                setGlobal('category', getGlobal('categories')[$i]);
+                print "<li>";
+                getTemplate('components/header/category-item');
+                print "</li>";
+              }
+            ?>
+          <?php else : ?>
+            <h2 class>No se encontraron categorías destacadas</h2>
+          <?php endif ?>
```
Comment (Miguelslo27) [inline template_dev/components/footer/footer.php]: Remove this `else` so if there are no categories, the user will see nothing, we should use that check to change the parent link of the dropdown, so if the dropdown has no items, should not be a dropdown at all. Make sense?

File: template_dev/components/header/header.php [hunk 3]
```diff
@@ -22,8 +22,19 @@
...
+                  setGlobal('category', $category);
+                  getTemplate('components/header/category-item')
+                ?>
+              <?php endforeach ?>
+            <?php else : ?>
+              <h2 class="not-found-cat">No se encontraron categorías destacadas</h2>
+              <?php if (isAdmin()) : ?>
+                  <a href="/categoria/nueva">Nueva categoría +</a>
+              <?php endif ?>
+            <?php endif ?>
```
Comment (Miguelslo27) [inline template_dev/components/header/header.php]: The same thing that the previous comment, in the footer we should not show a message if there are no categories, instead we should not show anything

File: template_dev/pages/category.php [hunk 4]
```diff
@@ -21,13 +21,15 @@
       exit;
     }
 
-    $categories = getCategories('`category_id` = 0 AND `status` = 1');
+    $categoriesTotal = getCategories('`category_id` = 0 AND `status` = 1');
     $where      = '`category_id` = ' . $currentCategory->id . ' AND `status` = 1';
     $pager      = getPager('articles', $where, ARTICLES_PER_PAGE);
     $articles   = getArticles($where, $pager->offset, $pager->per_page);
+    $categories            = getCategories();
```
Comment (Miguelslo27) [inline template_dev/pages/category.php]: Let's match the spaces between the variable and before the `=` sign

**Review by Miguelslo27 — CHANGES_REQUESTED** [review]
For some reason, when you **get into** a **category** page and see articles, the **sidebar** with all the other **categories is missing**. We should get the sidebar back to have a more comfortable navigation

**Comment by fdsosa** [comment]
> For some reason, when you **get into** a **category** page and see articles, the **sidebar** with all the other **categories is missing**. We should get the sidebar back to have a more comfortable navigation

El **sidebar** estaba dentro de una condicional donde había una variable que estaba definida en **articles**, la variable llama a la **categoria** con **id 0**, la cual no exisite por eso no se desplegaba el **sidebar**.

**Review by Miguelslo27 — APPROVED** [review]

**Pull request merged**